cmd/fund/fund.go

```diff
@@ -0,0 +1,132 @@
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, wallet := range wallets {
+		_, err := web3Client.Eth.SyncSendRawTransaction(
+			common.HexToAddress((*wallet).Hex()),
+			amountWei,
+			nonce,
+			walletFundingGas,
+			web3Client.Utils.ToGWei(1),
+			nil,
+		)
+		if err != nil {
+			log.Fatal(err)
+		}
+		fmt.Println("Funded", (*wallet).Hex(), "with", amountWei, "wei")
+		nonce++
+	}
+	return nil
+}
+
+// fundCmd represents the fund command
+var FundCmd = &cobra.Command{
+	Use:   "fund",
+	Short: "Bulk fund many crypto wallets automatically.",
+	Long:  usage,
+	Run: func(cmd *cobra.Command, args []string) {
+		fundingErr := runFunding(cmd)
+		if fundingErr != nil {
+			cmd.PrintErrf("Error funding wallets")
+		}
+	},
+}
+
+func runFunding(cmd *cobra.Command) error {
+	// setup new web3 session with remote rpc node
+	web3Client, clientErr := web3.NewWeb3(chainRPC)
+	if clientErr != nil {
+		cmd.PrintErrf("There was an error creating web3 client: %s", clientErr.Error())
+		return clientErr
+	}
+
+	// add pk to session for sending signed transactions
+	if setAcctErr := web3Client.Eth.SetAccount(fundingWalletPK); setAcctErr != nil {
+		cmd.PrintErrf("There was an error setting account with pk: %s", setAcctErr.Error())
+		return setAcctErr
+	}
+
+	// set proper chainId for corresponding chainRPC
+	cdkChainId := int64(chainID)
+	web3Client.Eth.SetChainId(cdkChainId)
+
+	// convert wallet address and pk format for downstream processing
+	fundingWalletAddressParsed := common.HexToAddress(fundingWalletAddress)
+	fundingWalletECDSA, ecdsaErr := crypto.HexToECDSA(fundingWalletPK)
+	if ecdsaErr != nil {
+		cmd.PrintErrf("There was an error getting ECDSA: %s", ecdsaErr.Error())
+		return ecdsaErr
+	}
+
+	// generate set of new wallet addresses
+	addresses, genWalletErr := generateWalletAddresses(walletCount)
+	if genWalletErr != nil {
+		cmd.PrintErrf("There was an error generating wallet addresses: %s", genWalletErr.Error())
+		return genWalletErr
+	}
+
+	// fund all crypto wallets
+	fmt.Println("Starting to fund loadtest wallets...")
+	fundWalletErr := fundWallets(web3Client, addresses, fundingWalletAddressParsed, fundingWalletECDSA, big.NewInt(int64(walletFundingAmt*1e18)), uint64(walletFundingGas))
+	if fundWalletErr != nil {
+		cmd.PrintErrf("There was an error funding wallets: %s", fundWalletErr.Error())
+		return fundWalletErr
+	}
+
+	return nil
+}
+
+func init() {
+	FundCmd.Flags().IntVar(&walletCount, "wallet-count", 2, "Number of wallets to fund")
+	FundCmd.Flags().StringVar(&fundingWalletAddress, "funding-wallet-address", "", "Origin wallet that will be doing the funding")
+	FundCmd.Flags().StringVar(&fundingWalletPK, "funding-wallet-pk", "", "Corresponding private key for funding wallet address, ensure you remove leading 0x")
+	FundCmd.Flags().IntVar(&chainID, "chain-id", 100, "Blockchain network chain id")
+	FundCmd.Flags().StringVar(&chainRPC, "chain-rpc", "", "Blockchain RPC node endpoint for sending funding transactions")
```

Comment:
nit: I'd modify the description and default value to keep it aligned with what we have in other command such as `polycli loadtest`: https://github.com/maticnetwork/polygon-cli/blob/4f69f6e7f2381b4ed7faeca270e87eddab2f5f28/cmd/loadtest/app.go#L197

```suggestion
	FundCmd.Flags().StringVarP(&chainRPC, "rpc-url", "r", "http://localhost:8545", "The RPC endpoint url")
```

note: also update flag name in `usage.md` if you end up doing this :)